.github/workflows/on-push-to-main-branch.yml

```diff
@@ -21,3 +21,47 @@ jobs:
           project_type: sdk
           sdk_language: Kotlin
           dev_docs_slug: kotlin
+
+      - name: Update release branch
+        uses: google-github-actions/release-please-action@v4
+        id: release
+        with:
+          token: ${{ secrets.MOMENTO_MACHINE_USER_GITHUB_TOKEN }}
+          path: src
+          release-type: java
+          package-name: client-sdk-kotlin
+          changelog-types: '[{"type":"feat","section":"Features","hidden":false},{"type":"fix","section":"Bug Fixes","hidden":false},{"type":"chore","section":"Miscellaneous","hidden":false}]'
+          extra-files: |
+            build.gradle.kts
+
+      - name: Set up JDK 17
+        uses: actions/setup-java@v4
+        with:
+          java-version: 17
+          distribution: 'corretto'
+        if: ${{ steps.release.outputs.src--release_created }}
```

Comment:
nit: you can make everything after the release please action a new job. Then the job could depend on the `if: ${{ steps.release.outputs.src--release_created }}`, and wouldn't need to copy this if state in every branch. I am just thinking in the future if we add any steps it could be something that is easy to miss

https://github.com/momentohq/client-sdk-dart/blob/790042304679b1deb218220b208188f5289fc73d/.github/workflows/release-please.yml#L12-L33

Reply:
Good idea.